api/desecapi/migrations/0012_domain_replicated.py

```diff
@@ -1,18 +0,0 @@
-# Generated by Django 3.1.4 on 2021-01-02 15:24
```

Comment:
belongs to 2f0dd672e5174ffafaa028fbf4a0013914907a70, but I guess we're going to squash that anyways